app/core/model/account/account.go

```diff
@@ -0,0 +1,56 @@
+package account
+
+import (
+	"stonehenge/app/core/types/audits"
+	"stonehenge/app/core/types/currency"
+	"stonehenge/app/core/types/document"
+	"stonehenge/app/core/types/id"
+	"stonehenge/app/core/types/password"
+)
+
+// Account holds useful information about accounts
+type Account struct {
+
+	// The unique identifier of this entity
+	Id id.ID `json:"id"`
```

Comment:
Seguindo boas [práticas do Go](https://github.com/golang/go/wiki/CodeReviewComments#initialisms), acredito que aqui poderia seguir a regra de consistencia de maiuscula ou minuscula em abreviações. 

```suggestion
	ID id.ID `json:"id"`
```

Reply:
Beleza Bella! Obrigado pelo apontamento, vou alterar as propriedades IDs